packages/earth-admin/src/pages-client/layers/home.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import { useAuth0 } from 'auth/auth0';
+import { AuthzGuards } from 'auth/permissions';
+
+import { ContentLayout } from 'layouts';
+import { Card, LinkWithOrg } from 'components';
```

Comment:
Please make component imports more explicit by not importing directly from components folder